.circleci/configurations/jobs.yml

```diff
@@ -95,7 +95,9 @@ jobs:
       # Run JavaScript tests
       - run:
           name: "Run Tests: JavaScript Tests"
-          command: node ./scripts/run-ci-javascript-tests.js --maxWorkers 2
+          command: |
+            yarn build
+            node ./scripts/run-ci-javascript-tests.js --maxWorkers 2
```

Comment:
Should be unnecessary once these package entry points are configured to run from source.